Frameworks/Foundation/NSDictionary.mm

```diff
@@ -99,8 +99,11 @@ - (NSArray*)allObjects {
         return nil;
     }
 
-    StrongId<NSArray> keyArray = [_keyEnumerator allObjects];
-    StrongId<NSMutableArray> toRet = [NSMutableArray new];
+    StrongId<NSArray> keyArray;
+    keyArray.attach([_keyEnumerator allObjects]);
+
+    StrongId<NSMutableArray> toRet;
```

Comment:
True. I went with the assumption of creator ownership as per ios documentation, given this class creates it and it's not the known (copy,alloc) methods. it's naturally autoreleased. 

---
In reply to: [73974242](https://github.com/Microsoft/WinObjC/pull/851#discussion_r73974242) [](ancestors = 73974242,73973850,73973752,73973293)